pkg/grpc/api_test.go

```diff
@@ -3062,39 +3062,7 @@ func TestStateAPIWithResiliency(t *testing.T) {
 			Keys:      []string{"failingBulkGetKey", "goodBulkGetKey"},
 		})
 
-		assert.NoError(t, err)
-		assert.Equal(t, 2, failingStore.Failure.CallCount("failingBulkGetKey"))
-		assert.Equal(t, 1, failingStore.Failure.CallCount("goodBulkGetKey"))
-	})
-
-	t.Run("bulk state get times out on single with resiliency", func(t *testing.T) {
```

Comment:
Because BulkGet is now implemented in components-contrib, it's not aware of item-specific resiliency. It will retry all GET operations at once. This is necessary to make sure that components can (although don't have to) implement a "native" BulkGet that performs a single round-trip to the database (for example, SQL databases now can perform a single "SELECT" query with multiple keys, which was not possible before).